sphinxext/opengraph/__init__.py

```diff
@@ -29,6 +29,8 @@
 
 
 def make_tag(property: str, content: str) -> str:
+    # Parse quotation, so they won't break html tags if smart quotes are disabled
+    content = content.replace('"', "&quot;")
```

Comment:
This should be in a separate PR.

Reply:
This was in a separate PR, it was moved so it would work with tags that come from field lists.